src/Entity/HealthCheckConfig.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
```

Comment:
Please don't forget to update license headers

Reply:
:+1: Bien vu